examples/plot_05_train_test_split.py

```diff
@@ -138,39 +143,48 @@
 X_train, X_test, y_train, y_test = skore.train_test_split(
     X=X, y=y, test_size=0.2, random_state=0
 )
-print(X_train, "\n", y_train, "\n", X_test, "\n", y_test)
+X_train_explicit = X_train.copy()
 
 # %%
 # Moreover, when passing ``X`` and ``y`` explicitly, the ``X``'s are always returned
 # before the ``y``'s, even when they are inverted:
 
 # %%
-arr = np.arange(10).reshape((5, 2))
+arr = X.copy()
 arr_train, arr_test, X_train, X_test, y_train, y_test = skore.train_test_split(
     arr, y=y, X=X, test_size=0.2, random_state=0
 )
-print(X_train, "\n", y_train, "\n", X_test, "\n", y_test)
+X_train_explicit_inverted = X_train.copy()
+
+print("When expliciting, with the small typo, are the `X_train`'s still the same?")
+print(np.allclose(X_train_explicit, X_train_explicit_inverted))
+
 
 # %%
 # Automatic diagnostics: raising methodological warnings
 # ------------------------------------------------------
 #
+# In this section, we show how skore can provide methodological checks.
+#
+# Class-imbalance
```

Comment:
```suggestion
# Class imbalance
```

Reply:
Plus all other instances of "class-imbalance"

Reply:
Done